internal/pkg/api/gitops.go

```diff
@@ -39,6 +40,13 @@ func WithFluxRepository(repository string) GitOpsConfigOpt {
 	}
 }
 
+func WithFluxRepositorySuffix(suffix string) GitOpsConfigOpt {
+	return func(c *v1alpha1.GitOpsConfig) {
+		repository := c.Spec.Flux.Github.Repository
+		c.Spec.Flux.Github.Repository = fmt.Sprintf("%s-%s", repository, suffix)
```

Comment:
I feel like this could be implemented where needed, by just using `WithFluxRepository`, it doesn't need to be in this module